src/main.js

```diff
@@ -1,36 +1,48 @@
 // Create variables targetting the relevant DOM elements here 👇
-var randomCover = document.querySelector(".random-cover-button");
-var saveCover = document.querySelector(".save-cover-button");
-var viewSaved = document.querySelector(".view-saved-button");
-var makeNewCover = document.querySelector(".make-new-button");
-var homeButton = document.querySelector(".home-button");
-
-var coverImage = document.querySelector(".cover-image")
-var coverTitle = document.querySelector(".cover-title")
-var firstDescriptor = document.querySelector(".tagline-1")
-var secondDescriptor = document.querySelector(".tagline-2")
-
-var homeView = document.querySelector(".home-view");
-var formView = document.querySelector(".form-view");
-var savedView = document.querySelector(".saved-view");
-
-
+var randomCoverButton = document.querySelector('.random-cover-button');
+var saveCoverButton = document.querySelector('.save-cover-button');
+var viewSavedButton = document.querySelector('.view-saved-button');
+var makeCoverButton = document.querySelector('.make-new-button');
+var homeButton = document.querySelector('.home-button');
+var makeBookButton = document.querySelector('.create-new-book-button');
```

Comment:
Awesome job refactoring the button variables to be more consistent and easy to read!